src/whyDidYouRender.js

```diff
@@ -90,6 +93,18 @@ export default function whyDidYouRender(React, userOptions){
   const origCreateFactory = React.createFactory
 
   let componentsMap = new WeakMap()
+  const ownerDataMap = new WeakMap()
+  const hooksRef = {current: []}
+
+  Object.defineProperty(React.__SECRET_INTERNALS_DO_NOT_USE_OR_YOU_WILL_BE_FIRED.ReactCurrentOwner, 'current', {
```

Comment:
let's use a more definite name for this one with WDYR prefix

Reply:
This is what React itself uses to track current owner, I just intercept the writes

Reply:
oh nice. i got it. please add a comment about it on top of it.
something like
```
// intercepting assigns to ReactCurrentOwner.current
```